app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -97,7 +97,7 @@ def queue_name_for_ems(ems)
       return "generic" if ems.kind_of?(Host) && ems.acts_as_ems?
 
       return ems unless ems.kind_of?(ExtManagementSystem)
```

Comment:
yes, and its called from https://github.com/ManageIQ/manageiq/blob/bf9868c8697396cf917413facf2c9a681f4496fe/app/models/mixins/per_ems_worker_mixin.rb#L46-L46 with queue_name.

TBH the concern should be `start_worker_for_ems` and in `stop_worker_for_ems` and not in `queue_name_for_ems` - but somebody thought it would be DRY'er to do it this way back in the days 😄 

@jrafanie do you want this to be refactored?